modules/rabbitmq/src/main/java/org/testcontainers/containers/RabbitMQContainer.java

```diff
@@ -0,0 +1,358 @@
+package org.testcontainers.containers;
+
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import org.jetbrains.annotations.NotNull;
+import org.json.JSONObject;
+import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
+import org.testcontainers.containers.wait.strategy.WaitAllStrategy;
+import org.testcontainers.containers.wait.strategy.WaitStrategy;
+import org.testcontainers.utility.MountableFile;
+
+import java.io.IOException;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Stream;
+
+import static java.lang.String.join;
+import static java.util.Arrays.asList;
+import static java.util.Collections.emptyList;
+import static java.util.stream.Collectors.toSet;
+
+/**
+ * Testcontainer for RabbitMQ.
+ *
+ * @author Martin Greber
+ */
+public class RabbitMQContainer<S extends RabbitMQContainer<S>> extends GenericContainer<S> {
+
+    /**
+     * The image defaults to the official RabbitmQ image: <a href="https://hub.docker.com/_/rabbitmq/">RabbitMQ</a>.
+     */
+    private static final String DEFAULT_IMAGE_NAME = "rabbitmq";
+    private static final String DEFAULT_TAG = "3.7-management-alpine";
+
+    private static final int DEFAULT_AMQP_PORT = 5672;
+    private static final int DEFAULT_AMQPS_PORT = 5671;
+    private static final int DEFAULT_HTTPS_PORT = 15671;
+    private static final int DEFAULT_HTTP_PORT = 15672;
+
+    private String adminPassword = "guest";
+    private String adminUsername = "guest";
+    private List<String> plugins = emptyList();
+    private List<List<String>> values = new ArrayList<>();
```

Comment:
```suggestion
    private final List<List<String>> values = new ArrayList<>();
```

Reply:
Also, can we adjust the naming of this variable? After reviewing the impl, it feels more like `postStartCommands` or something